src/commands/actions/arguments/owner-nonce.ts

```diff
@@ -4,7 +4,7 @@ export default {
   options: {
     type: Number,
     required: true,
-    help: 'The validator owner nonce of the account (owner address) within the SSV contract (increments after each validator registration), obtained using the ssv-scanner tool.'
+    help: 'The validator registration nonce of the account (owner address) within the SSV contract (increments after each validator registration), obtained using the ssv-scanner tool.'
```

Comment:
The same is for all the `help` options, some of them I see without dots and some - with